testsuite/run_sets/build_validation/build_validation_init_salt_migration.yml

```diff
@@ -0,0 +1,7 @@
+# This file describes the order of features in a Build Validation test suite run.
```

Comment:
I would prefer to keep salt_migration in migration rake because we don't only init the salt migration client in this feature.